src/vivarium_ciff_sam/constants/data_values.py

```diff
@@ -18,7 +18,7 @@
 
 # diarrhea duration in days
 DIARRHEA_DURATION: Tuple = (
-    'diarrheal_diseases_duration', get_norm_from_quantiles(mean=4.04485, lower=3.94472, upper=4.144975)
+    'diarrheal_diseases_duration', get_norm_from_quantiles(mean=4.3, lower=4.3, upper=4.3)
```

Comment:
This used to have a lower and upper of 4.2 and 4.5, i think. do we want this no-uncertainty distribution? i'm not seeing reference to this on slack

Reply:
Hmm they said they wanted 4.3 again, but perhaps they actually wanted the old distribution. Can check in next week if we are doing more runs then